packages/pyre-ast/pyre-ast.0.1.2/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Full-fidelity Python parser in OCaml"
+description:
+  "pyre-ast is an OCaml library to parse Python source files into abstract syntax trees. Under the hood, it relies on the CPython parser to do the parsing work and therefore the result is always 100% compatible with the official CPython implementation."
+maintainer: ["grievejia@gmail.com"]
+authors: ["Jia Chen"]
+license: "MIT"
+homepage: "https://github.com/grievejia/pyre-ast"
+doc: "https://grievejia.github.io/pyre-ast/doc"
+bug-reports: "https://github.com/grievejia/pyre-ast/issues"
+depends: [
+  "dune" {>= "2.8" & >= "2.7"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
To get rid of this for good, you can remove dune from the list of dependencies in the dune-project file. Dune will add itself automatically to the list, with the correct lower bound